src/main/app/src/app/formulieren/taken/abstract-taak-formulier.ts

```diff
@@ -142,13 +143,25 @@ export abstract class AbstractTaakFormulier {
             this.dataElementen = {};
         }
         Object.keys(formGroup.controls).forEach((key) => {
-            if (key !== AbstractTaakFormulier.TOEKENNING_FIELD) {
+            if (key !== AbstractTaakFormulier.TOEKENNING_FIELD && !this.isReadonlyFormField(key)) {
                 this.dataElementen[key] = formGroup.controls[key]?.value;
             }
         });
         return this.dataElementen;
     }
 
+    private isReadonlyFormField(id: string): boolean {
```

Comment:
je kan ook een for-of gebruiken
private isReadonlyFormField(id: string): boolean { for (const fieldArray of this.form) { for (const field of fieldArray) { if (field.id === id) { return formField.readonly; } } } throw new Error(FormField: "${id}" not found!); }